graphrag/index/graph/extractors/community_reports/sort_context.py

```diff
@@ -154,3 +154,81 @@ def _get_context_string(
         )
 
     return context_string
+
+
+def sort_context_batch(
+    local_contexts: pd.DataFrame,
+    node_details_column: str = schemas.NODE_DETAILS,
+    edge_details_column: str = schemas.EDGE_DETAILS,
+    claim_details_column: str = schemas.CLAIM_DETAILS,
+    community_id_column: str = schemas.COMMUNITY_ID,
+    max_tokens: int | None = None,
+) -> pd.DataFrame:
+    """Batch processing for community context strings."""
+
+    def generate_context(group):
+        # Explode and deduplicate edges and claims
+        edges = pd.DataFrame(
+            group[edge_details_column].explode().dropna().tolist()
+        ).drop_duplicates()
+        claims = pd.DataFrame(
+            group[claim_details_column].dropna().tolist()
+        ).drop_duplicates()
+        nodes = pd.DataFrame(
+            group[node_details_column].dropna().tolist()
+        ).drop_duplicates()
+
+        # Sort dataframes for consistent output
+        if not nodes.empty:
```

Comment:
what happens if this is empty and we sort it? does it fails? could we just sort and remove the `if not empty` statements?